build.gradle

```diff
@@ -119,7 +119,7 @@ subprojects {
                       'com.fasterxml.jackson.core:jackson-databind:2.6.3',
                       'com.fasterxml.jackson.core:jackson-annotations:2.6.3'],
       jadira_usertype: 'org.jadira.usertype:usertype.core:4.0.0.GA',
-      jcommander:    'com.beust:jcommander:1.35',
+      jcommander:    'com.beust:jcommander:1.72',
```

Comment:
Are all these really needed or are these just cut and paste from the main build.gradle?

Reply:
This is the main build.gradle, I changed the version number of JCommander to address a compile error I was getting in tcg_rim_rool.